### PR TITLE
(DomActions) WTR tests for elementVisible()

### PR DESCRIPTION
**Pull request by freiondrej**

### CODE REVIEW

File: tests-wtr/dom-actions/dom-actions.element-visible.ts
Comment (freiondrej) [inline tests-wtr/dom-actions/dom-actions.element-visible.ts:1]: This test is quite repetitive and I was considering some kind of matrix approach, but ultimately didn't do it as it seems to me the test cases are so short and concise, the maintainability might actually suffer if we try to DRY it. Let me know what you think 🙏

Comment (muodov) [reply tests-wtr/dom-actions/dom-actions.element-visible.ts:1]: I don't mind either way. You could also reuse the same `domActions` instance across multiple tests, then the tests would be basically one-liners.

File: tests-wtr/dom-actions/dom-actions.element-visible.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<body>
+<script type="module">
+  import {runTests} from '@web/test-runner-mocha';
+
+  runTests(async () => {
+    await import('./dom-actions.element-visible')
+  });
+</script>
+<div id="none-visible">
+    <button style="display:none">This one is invisible</button>
```
Comment (muodov) [inline tests-wtr/dom-actions/dom-actions.element-visible.html:11]: Could you also add a case when a button element is not hidden itself, but its parent is?
```html
<div style="display:none">
  <button>I am hidden too</button>
</div>
```
There are many more possible cases, but these are probably most common.

**Review by muodov — CHANGES_REQUESTED** [review]

File: tests-wtr/dom-actions/dom-actions.element-visible.ts
Comment (freiondrej) [reply tests-wtr/dom-actions/dom-actions.element-visible.ts:1]: uuu nice, I quite like that! The one-liners are pretty elegant 🚀 👏

File: tests-wtr/dom-actions/dom-actions.element-visible.html (hunk 1 above)
Comment (freiondrej) [reply tests-wtr/dom-actions/dom-actions.element-visible.html:11]: Do you mean add it as simply as [this](23b3368ddf420404e770c5f484b601e715b46f59)?

**Review by muodov — APPROVED** [review]
LGTM! thanks again!

**Pull request merged**